open-apparel-registry/open-apparel-registry
# Replace custom ECS management command with django-ecsmanage

## Overview

- Remove custom ECS management command
- Add support for [django-ecsmanage](https://github.com/azavea/django-ecsmanage)

Resolves #470 

### Checklist


## Notes

This PR introduces a slight change in behavior for the `ecsmanage` command. Before, you'd have to pass `-e staging` to target staging. Now, staging is the default environment, and you only need to pass `-e production` if you'd like to target production.

## Testing Instructions

Kick off an ECS management command in `staging`:

```bash
[rocky@algiers open-apparel-registry] (feature/jrb/django-ecsmanage)$ ./scripts/manage ecsmanage showmigrations
Starting open-apparel-registry_database_1 ... done
INFO:botocore.credentials:Found credentials in shared credentials file: ~/.aws/credentials
Task started! View here:
https://console.aws.amazon.com/ecs/home?region=eu-west-1#/clusters/ecsStagingCluster/tasks/6284545a-d9f4-4bfa-b4de-d1437334b8f7/details
```

Kick off an ECS management command in `production`:

```bash
[rocky@algiers open-apparel-registry] (feature/jrb/django-ecsmanage)$ ./scripts/manage ecsmanage -e production showmigrations
Starting open-apparel-registry_database_1 ... done
INFO:botocore.credentials:Found credentials in shared credentials file: ~/.aws/credentials
Task started! View here:
https://console.aws.amazon.com/ecs/home?region=eu-west-1#/clusters/ecsProductionCluster/tasks/332a61be-2feb-44bc-bcc0-bb4cb72d724c/details
```

Observe an exit code of `0`. Unfortunately, the lack of action from AWS on #358 means I haven't been able to see any log output.
